tools/assetpipelineeditor/mainwindow.cpp

```diff
@@ -362,12 +362,18 @@ void MainWindow::compressFile()
         Kuesa::GLTF2Exporter exporter;
         exporter.setContext(importer->context());
         exporter.setScene(m_entity);
-        rootObject = exporter.compress(QFileInfo(gltfFile).dir(), rootObject);
+
+        auto srcdir = QFileInfo(gltfFile).dir();
+        srcdir.mkdir("compressed");
+
+        auto targetdir = srcdir;
```

Comment:
const auto targetdir = srcdir.cd("compressed");

Reply:
does not exist anymore